extensions/json-format/package.json

```diff
@@ -32,6 +33,12 @@
       "description": "Formats a JSON/JS Object string selected in the foremost editor and copies/pastes it back",
       "mode": "no-view"
     },
+    {
+      "name": "formatSelectionAndShow",
+      "title": "Format Selected JSON and Show",
+      "description": "Formats a JSON/JS Object string selected in the foremost editor and shows it in the viewer",
```

Comment:
Json/JS ?

Reply:
I imitated several other commands, and the other commands were written in JSON/JS.